config.js

```diff
@@ -36,6 +36,12 @@ config.mqtt = {
      */
```

Comment:
CHANGES_NEXT_RELEASE entry about the support to HTTPS should be added.

In addition, I understand this PR provides MQTTS support so this issue would be closed by the PR https://github.com/telefonicaid/iotagent-ul/issues/354, wouldn't it? In that case, an entry about that should be also added about it.

Reply:
Fixed 146025b

Reply:
CNR updated to refer to  #354 - Fixed eb71a30